game-core/src/main/java/games/strategy/engine/framework/GameRunner.java

```diff
@@ -153,7 +156,12 @@ private static void loadGame() {
     gameSelectorModel.loadDefaultGameSameThread();
     final String fileName = System.getProperty(TRIPLEA_GAME, "");
     if (!fileName.isEmpty() && new File(fileName).exists()) {
-      gameSelectorModel.load(new File(fileName));
+      try {
+        gameSelectorModel.load(new File(fileName));
+      } catch (final Exception e) {
+        log.log(
+            Level.SEVERE, "Error loading game file: " + new File(fileName).getAbsolutePath(), e);
```

Comment:
`new File(fileName)` is called three times here, I'd appreciate if it was put into a single variable and used 3 times, even if that adds another if block

Reply:
This one

Reply:
The only reason I'm pointing this out is because this PR introduces another call to `new File(fileName)`.
So I was thinking that now that there's 3 of them adding a variable and use that3 times would finally be worth it, but nothing too crazy here. I can also do that in a follow-up PR. It's nothing special

Reply:
It is your prerogative. IMO it could be more effective prioritization to focus on 2.0 or deeper refactoring that would remove game loading from `GameRunner`

Reply:
Removing this bit of the code would be a good step into the right direction, but a hard one to achieve  unfortunately.

The "join headless host" code depends on it, so it can pass parameters to the newly spawned process 😅